astitodo/main.go

```diff
@@ -34,7 +35,15 @@ func main() {
 
 		// Filter results for assignee
 		if *assignee != "" {
-			todos = todos.AssignedTo(*assignee)
+			assignees := strings.Split(*assignee, ",")
+
+			var filteredTODOs astitodo.TODOs
+
+			for _, v := range assignees {
```

Comment:
Instead of looping in assignees in the `main` package, I'd rather you update the `astitodo` package and change the signature:

```go
func (todos TODOs) AssignedTo(assignee string) (filteredTODOs TODOs)
```

to

```go
func (todos TODOs) AssignedTo(assignee ...string) (filteredTODOs TODOs)
```

That way the `AssignedTo` function can indeed take several input assignees which is a good idea and you can simply inject it `strings.Split(*assignee, ",")...`